src/graphorm/index.ts

```diff
@@ -247,10 +255,24 @@ const obj = new GraphORM({
     },
   },
   Post: {
-    additionalQuery: (ctx, alias, qb) =>
-      qb
+    additionalQuery: (ctx, alias, qb) => {
+      qb = qb
         .andWhere(`"${alias}"."deleted" = false`)
-        .andWhere(`"${alias}"."visible" = true`),
+        .andWhere(`"${alias}"."visible" = true`);
+
+      if (ctx.userId && ctx.isPlus) {
+        qb = qb
+          .innerJoin(Settings, 's', 's."userId" = :userId', {
+            userId: ctx.userId,
+          })
+          .addSelect(
+            `"s"."flags"->'clickbaitShieldEnabled'`,
+            'clickbaitShieldEnabled',
+          );
+      }
```

Comment:
This will do inner join for all plus user post queries, which could affect performance, I think we should query only when needed or expose it on context.

Reply:
This was the only way I figured out how to do it in GraphORM 🙈

Reply:
As discussed, we'll leave it as is for now, and I'll circle back to this later to see if there's any improvements that can be done.